### DISCUSSION TITLE
在倉頡打無效的碼按空格不要出其英文字串

### DISCUSSION THREAD

**Discussion by 307587** [rime/home#1402] [Q&A]
打了沒對應漢字的倉頡碼按空格就出英文要退回去很礙事，如何取消這功能？

**Comment by Sunpulseqd** [comment]
- {accept: space, send: Escape, when: composing} # 空码按空格清空

**Reply by 307587** [reply]
這要加在哪？

**Reply by demon-Dark** [reply]
default.custom.yaml文件里的key_binder:下面的bindings:
https://github.com/rime/home/wiki/CustomizationGuide#%E7%A9%BA%E7%A2%BC%E6%99%82%E6%8C%89%E7%A9%BA%E6%A0%BC%E9%8D%B5%E6%B8%85%E7%A9%BA%E8%BC%B8%E5%85%A5%E7%A2%BC

**Reply by 307587** [reply]
爲何加了、部署了也沒用？

**Reply by demon-Dark** [reply]
那就加到你输入法方案里
![e53d99e5c6b5e55feb6fffa311fff02](https://github.com/rime/home/assets/134026135/9c87783b-8364-48a8-b007-706348ae2f6d)
